scripts/prepublish

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Suggest adding `.sh` extension to all bash scripts.

Reply:
done